packages/beacon-node/src/metrics/validatorMonitor.ts

```diff
@@ -287,6 +294,39 @@ export function createValidatorMonitor(
       }
     },
 
+    registerValidatorStatuses(currentEpoch, status, index, balance) {
+      // Prevent registering status for the same epoch twice. processEpoch() may be ran more than once for the same epoch.
+      if (currentEpoch <= lastRegisteredStatusEpoch) {
+        return;
+      }
+      lastRegisteredStatusEpoch = currentEpoch;
+
+      switch (status) {
+        case "active":
+        case "active_exiting":
+        case "active_slashed":
+        case "active_ongoing":
+          metrics.validatorMonitor.validatorStatus.inc({status: "active"});
+          break;
+        case "exited_unslashed":
+        case "withdrawal_done":
+          metrics.validatorMonitor.validatorStatus.inc({status: "exited"});
+          break;
+        case "pending_initialized":
+        case "pending_queued":
+          metrics.validatorMonitor.validatorStatus.inc({status: "pending"});
+          break;
+        case "withdrawal_possible":
+          metrics.validatorMonitor.validatorStatus.inc({status: "withdrawalable"});
```

Comment:
shouldn't these be the same as the simple validator statuses (`SimpleValidatorStatus`)?

if we keep it as is, need to fix typo

```suggestion
          metrics.validatorMonitor.validatorStatus.inc({status: "withdrawable"});
```